web/themes/contrib/civictheme/src/CivicthemeConstants.php

```diff
@@ -175,4 +175,15 @@ final class CivicthemeConstants {
    */
   const OPTOUT_VIEWS_STYLE_TABLE = 'CivicThemeOptoutViewsStyleTable';
 
+  /**
+   * Defines the image style constants.
+   */
+  const PROMO_IMAGE_STYLE = 'civictheme_promo_card';
```

Comment:
@AlexSkrypnyk We need to decide on which style to be used for each type of cards.